receiver/k8sclusterreceiver/internal/collection/nodes.go

```diff
@@ -34,14 +34,21 @@ const (
 	nodeCreationTime = "node.creation_timestamp"
 )
 
-func getMetricsForNode(node *corev1.Node, nodeConditionTypesToReport []string) []*resourceMetrics {
-	metrics := make([]*metricspb.Metric, len(nodeConditionTypesToReport))
+var allocatableDesciption = map[string]string{
+	"cpu":               "How many CPU cores remaining that the node can allocate to pods",
+	"memory":            "How many bytes of RAM memory remaining that the node can allocate to pods",
+	"ephemeral-storage": "How many bytes of ephemeral storage remaining that the node can allocate to pods",
+	"storage":           "How many bytes of storage remaining that the node can allocate to pods",
+}
 
-	for i, nodeConditionTypeValue := range nodeConditionTypesToReport {
+func getMetricsForNode(node *corev1.Node, nodeConditionTypesToReport, allocatableTypesToReport []string) []*resourceMetrics {
+	var metrics []*metricspb.Metric
+	//Condition
```

Comment:
These comments are not very useful. Please break down the function into two or improve comments according to [go guidelines](https://golang.org/doc/effective_go#commentary).

Reply:
Added appropriate comments